CoreFoundation/Base.subproj/CFInternal.h

```diff
@@ -349,7 +349,7 @@ CF_EXPORT void * __CFConstantStringClassReferencePtr;
 
 #if DEPLOYMENT_RUNTIME_SWIFT
 
-CF_EXPORT void *__CFConstantStringClassReference[];
+CF_EXPORT void *__CFConstantStringClassReference[] asm("_T010Foundation19_NSCFConstantStringCN");
```

Comment:
It would be nice if we had some way to avoid this disparity between the two when the mangling changes. Which seems to be a common occurrence... we had the same problem with swift 3.

Is this mangled name based off of the swift-4.0-branch names?